types/react-router-dom/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "dependencies": {
+    "@types/history": "*",
```

Comment:
`@types/history` needs to be pinned to `^4.7.2` as well because `History.Location` was not a generic type in older versions.

Reply:
DT implicitly uses the newest version of other DT packages.